js/libs/form.js

```diff
@@ -655,6 +655,34 @@ Fliplet.Widget.instance('form-builder', function(data) {
           $vm.saveProgress();
         }, 10);
 
+        function validateCheckboxValue(value, options) {
+          value = _.isArray(value) ? value : [value];
+
+          value = value.map(function(elem) {
+            return elem.trim();
+          });
+
+          if (options.length) {
+            var valueInOptions = [];
+
+            value.forEach(function(elem) {
+              if (_.find(options, { label: elem })) {
```

Comment:
I remember users can set values to a checkbox that is different from the label, e.g.

```
Option 1 <1>
```

...would produce an option that has the label `Option 1` and value `1`. In this case, we should be matching by the value, not by the label.

This line here suggests that we're matching by label.

Reply:
fixed